ecommerce/mail_api.py

```diff
@@ -0,0 +1,64 @@
+"""Ecommerce mail functions"""
+import pycountry
+import logging
+
+from ecommerce.messages import OrderReceiptMessage
+from mitol.mail.api import get_message_sender
+
+log = logging.getLogger()
+
+
+def send_ecommerce_order_receipt(order):
```

Comment:
Could you add a test for this? I don't think there's an example of comparable complexity on how to do this, but the gist of it would be mocking out `send_message` [like this](https://github.com/mitodl/ol-django/blob/main/tests/mitol/mail/test_api.py#L230), which should be passed an instance of your message class, created via its [`create()` method](https://github.com/mitodl/ol-django/blob/main/src/mitol/mail/messages.py#L43-L69). You'd just want to assert on the title and verify the contents of the body contain the textual content you're expecting.